presto-product-tests/src/main/resources/sql-tests/testcases/horology_functions/atTimeZoneAccessible.sql

```diff
@@ -1,3 +1,3 @@
 -- database: presto; groups: qe, horology_functions
-SELECT timezone_hour(TIMESTAMP '2001-08-22 03:04:05.321' at time zone 'Asia/Oral'),
-       timezone_minute(TIMESTAMP '2001-08-22 03:04:05.321' at time zone 'Asia/Oral')
\ No newline at end of file
+SELECT timezone_hour(TIMESTAMP '2001-08-22 03:04:05.321+00:00' at time zone 'Asia/Oral'),
```

Comment:
This semantic exception message `must be a time or timestamp with time zone` suggests that time zone suffix had to be added to the value. However, I don't know if it is a desired behaviour. If it is not, maybe lets create an issue in order to fix this and I will drop this pull request.